build/ks-apiserver/Dockerfile

```diff
@@ -1,12 +1,11 @@
 # Copyright 2020 The KubeSphere Authors. All rights reserved.
 # Use of this source code is governed by an Apache license
 # that can be found in the LICENSE file.
-FROM alpine/helm:3.4.2 as helm-base
+FROM openpitrix/helm:v0.1.0 as base
```

Comment:
Why change it? By the way, we cannot know the version of helm from image `openpitrix/helm:v0.1.0`. It's unclear.

Reply:
1. alpine/helm only has one `arm` version, which is alpine/helm:3.5.0-rc.1. So we beed build arm version ourself. [https://github.com/kubesphere/docker-helm/blob/main/Dockerfile#L4-L5](docker-helm) 
2. ks-controller-manager needs kustomize,  openpitrix/helm:v0.1.0 contains both helm and kustomize.

Reply:
yeah, agree with @LinuxSuRen we should set helm/kustomize version explicitly in Dockerfile, shouldn't depend on `openpitrix/helm:v0.1.0`, if not, what if the user wants to build an image with lastest helm/kustomize version?

Reply:
Neither helm nor kustomize has a suitable docker image (the community's current version does not support arm64), so we opened a separate project https://github.com/kubesphere/docker-helm to build the image we needed. helm and kustomize versions are defined in this file https://github.com/kubesphere/docker-helm/blob/main/Dockerfile#L4-L5